airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -307,6 +354,17 @@ def _build_lifecycle_config(self, cluster_data):
 
         return cluster_data
 
+    def _build_driver_pool(self):
+        driver_pool = {
+            "node_group": {
+                "roles": ["DRIVER"],
+                "node_group_config": {"num_instances": self.driver_pool_size},
```

Comment:
Thoughts on allowing instance type here as well? Something like:

```suggestion
                "node_group_config": {
                    "num_instances": self.driver_pool_size
                    "machine_type_uri": self.driver_pool_machine_type
                 },
```

Maybe this is too much config though? I'm fine with leaving this for another day.

Reply:
Let keep it these two for now. Later on, we can driver pool config (like InstanceFlexibilityPolicy) to cover rest of the config options.